<?xml version="1.0"?>
<launch>
    <!-- <node pkg="twist_controller" type="dbw_node.py" name="dbw_node" cwd="node" output="screen"> -->


    <node pkg="twist_controller" type="twist_controller" name="dbw_node" cwd="node" output="screen">
        <!-- vehicle weight  -->
        <param name="vehicle_mass" value="1736.35" />
        <param name="fuel_capacity" value="13.5" />
        <param name="brake_deadband" value=".1" />
        <param name="decel_limit" value="-3." />
        <param name="accel_limit" value="1." />
        <!-- 330 mm -->
        <param name="wheel_radius" value="0.2413" />
        <!-- m -->
        <param name="wheel_base" value="2.677" />
        <!-- steer / wheel -->
        <param name="steer_ratio" value="14.46" />
        <!-- lat acc: m/s^2 -->
        <param name="max_lat_accel" value="3." />
        <!-- the max steel angle 430 degree = 7.7 radius -->
        <param name="max_steer_angle" value="450.0" />
    </node>
</launch>
